### DISCUSSION TITLE
Dictate where pane attaches?

### DISCUSSION THREAD

**Discussion by kslxyz** [beakerbrowser/beaker#1881] [Q&A]
I'm using the Beaker Panes API to have links attach a pane rather than navigating away from my site or opening in a new tab. Currently the pane defaults to attach on the side, but for links leading to non-responsive sites, the half-window view breaks the desktop layouts. Is there a way for me to force the pane to attach to the bottom of the window instead?

**Comment by pfrazee** [comment]
Not at the moment, sorry. I'd be open to a PR which adds this.
